doc/dynamic_discovery_configuration_guide.md

```diff
@@ -134,31 +148,86 @@ Similarly, in the AS4 message that's sent the `/UserMessage/PartyInfo/To` elemen
         <ns:To>
         </ns:To>
     </ns:PartyInfo>
+    <ns:MessageProperties>
+        <ns:Property name="originalSender" type="urn:oasis:names:tc:ebcore:partyid-type:unregistered">C1</ns:Property>
+        <ns:Property name="finalRecipient" type="urn:oasis:names:tc:ebcore:partyid-type:unregistered">C4</ns:Property>
+    </ns:MessageProperties>
 ...
 <ns:UserMessage>
 ```
 
+**Note:** Also, the following elements in the PMode and AS4 message must match with the values defined by the receiving
+party (`C3`):
+
+* document identifier scheme;
```

Comment:
The list here should probably also use "-" as list item markers instead of "*" for consistency.

Reply:
Fixed.